src/actors.c

```diff
@@ -3392,6 +3393,7 @@ void func_802A0450(Player *player, struct Actor *actor) {
     }
 }
 
+// 0x802A0D54
```

Comment:
When the project is compiled. A map file is created in `build/us/mk64.us.map`. If you search for functions in the map file you can see what its address is. This makes it unnecessary to add addresses as comments to symbol in code. However, for .data/rodata you'll see me sometimes add addresses in those `.s` assembly files.